deps/Makefile

```diff
@@ -740,6 +732,9 @@ PCRE_OBJ_TARGET = $(build_shlibdir)/libpcre2-8.$(SHLIB_EXT)
 
 # Force optimization for PCRE flags (Issue #11668)
 PCRE_CFLAGS = -O3
+ifneq ($(OS),WINNT)
+PCRE_LDFLAGS = "-Wl,-rpath,'$(build_libdir)'"
```

Comment:
Actually, it's so that [Launchpad will shut up and build Julia](https://launchpadlibrarian.net/209478638/buildlog_ubuntu-utopic-amd64.julia_0.4.0-2362~ubuntu14.10.1_BUILDING.txt.gz).  ;)  Zoom to the end and look for the part where `dpkg-shlibdeps` very helpfully checks that every shlib has all its dependencies satisfied.